### DISCUSSION TITLE
Just a random rant

### DISCUSSION THREAD

**Discussion by adzubla** [nushell/nushell#5605] [Q&A]
Google always swaps "nushell" for "nutshell"!!!

**Comment by simonrouse9461** [comment]
I kind of like the name "nutshell" instead of "nushell". If it's "a NU(ew) Type of SHELL", why don't they just name it "Nutshell" without omitting the T? "nutshell" sounds much cuter in my opinion. They can even make the website heading to be "In a nutshell, it's a new type of shell."

**Reply by ghost** [reply]
i don't think looking "nutshell" up would be easy - rust is a way bigger project and constantly gets confused with the video game (which is a way less common term in programming context than nutshell in a shell context)

**Comment by 2brownc** [comment]
If you already don't know you can put nushell in quotes and google won't bother you.

Eg: `why is "nushell" so awesome` (google only suggests a correction once at the top that's it.)
